R/locations.R

```diff
@@ -12,6 +12,10 @@
 #' @template radius
 #' @template limit
 #' @template page
+#' @template siteType
```

Comment:
I wonder whether we should make all param names "snake case" although the API params are "camel case":

Reply:
hm good question. I really like the "camel case" version, but I don't have enough experience at all to judge about this. I think however that `activation_date_from` should better be `activationDate_from` and `activationDate_to` right?